velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1096,6 +1096,31 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct CurrentTimeStampFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  std::optional<int64_t> sessionTzID_;
+
+  FOLLY_ALWAYS_INLINE void initialize(const core::QueryConfig& config) {
+    auto sessionTzName = config.sessionTimezone();
+    if (!sessionTzName.empty()) {
+      sessionTzID_ = util::getTimeZoneID(sessionTzName);
+    }
+  }
+
+  FOLLY_ALWAYS_INLINE void call(out_type<TimestampWithTimezone>& result) {
+    int64_t current_timestamp = Timestamp::now().toMillis();
+
+    std::optional<int64_t> sessionTzID_;
```

Comment:
This variable definition hides the member variable in the struct. Is this intended ?

Reply:
Oh, this should be a duplicated definition. I will remove it.

Reply:
Np. My bad. I didn't see the second use.